### DISCUSSION TITLE
Tetthet: Comfortable

### DISCUSSION THREAD

**Discussion by Stakset** [fremtind/jokul#3129] [Spørsmål] [answered]
Hvorfor heter den ene størrelsen Comfortable på Jøkul portalen? Burde kanskje vært noe som hang mer sammen med komponentene i Figma? Desktop eller default f.eks?

**Comment by wkillerud** [comment]
Kan sikkert gå for Default i stedet om det gir mer mening, eller bare en Checkbox for å skru av og på compact.

I kode er compact/ikke-compact og desktop/mobil to ulike "spaker", så vi trengte et navn i kode for motsatsen til Compact. Var vanskelig å komme på noe godt navn, så vi stjal like greit Comfortable/Behagelig fra Gmail 😄

**Comment by onordbo** [comment]
Da vi pusset opp compact ble det gjort litt ekstra "rørleggerarbeid" for skalerbarhetens skyld. Om det skaper unødig forvirring kan vi pakke eksemplene inn i en checkbox, som @wkillerud nevner.

**Comment by Murstam** [comment, marked as answer]
Men ja, hvis det skal ha et navn tenker jeg default, normal, standard er enklere å forstå.

**Reply by Stakset** [reply]
Default fungerer fint :)

**Reply by wkillerud** [reply]
Endret til Default i dag
